src/error/mod.rs

```diff
@@ -91,3 +94,41 @@ impl Display for OpenTsConfigError {
         }
     }
 }
+
+#[derive(Debug, Serialize)]
+pub struct JsonErrorFile<'a> {
+    pub violations: Vec<ImportRuleViolation<'a, 'a>>,
+}
+
+pub fn write_errors_as_json(
```

Comment:
Violations and errors are separate concepts, this doesn't belong in `error`.

I don't see this needing to be on the js api surface, so this can be part of the cli module.